Scheduler service not working

Logs:
```
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /usr/src/code/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/usr/src/code/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-04-09T06_54_37_385Z-debug.log
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -16,6 +16,8 @@ services:
     image: node:12.13-alpine
     working_dir: /usr/src/code
     command: ls -la
+    depends_on:
+      - "web"
     environment:
       - NODE_ENV
     volumes:

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -11,6 +11,7 @@ services:
       - '${DOCKER_WEB_PORT:-8080}:${PORT:-8080}'
     volumes:
       - app:/usr/src/code
+
   scheduler:
     image: node:12.13-alpine
     working_dir: /usr/src/code

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -15,7 +15,7 @@ services:
   scheduler:
     image: node:12.13-alpine
     working_dir: /usr/src/code
-    command: npm run start:scheduler
+    command: ls -la
     environment:
       - NODE_ENV
     volumes:

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -15,7 +15,7 @@ services:
   scheduler:
     image: node:12.13-alpine
     working_dir: /usr/src/code
-    command: ls -la
+    command: npm run start:scheduler
     depends_on:
       - "web"
     environment:

